Tests/WebSocketKitTests/WebSocketKitTests.swift

```diff
@@ -547,12 +547,12 @@ final class WebSocketKitTests: XCTestCase {
     }
     
     func testCreateNewELGAndShutdown() throws {
-        let client = WebSocketClient(eventLoopGroupProvider: .createNew)
+        let client = WebSocketClient(eventLoopGroupProvider: .shared(.singletonMultiThreadedEventLoopGroup))
```

Comment:
Did this have a warning or something? because the test name doesn't quite match the change you made.